x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/ReloadRemoteClusterCredentialsIT.java

```diff
@@ -1,314 +0,0 @@
-/*
```

Comment:
This is entirely superseded by `RemoteClusterSecurityReloadCredentialsRestIT` which I was able to add with https://github.com/elastic/elasticsearch/pull/103133 merged. I think the maintenance burden justifies removing the old test suite.